can-stache-bindings.js

```diff
@@ -1202,9 +1208,20 @@ var decodeAttrName = function(name){
 var makeDataBinding = function(node, el, bindingData) {
 	// Get information about the binding.
 	var bindingInfo = getBindingInfo(node, bindingData.attributeViewModelBindings, bindingData.templateType, el.nodeName.toLowerCase(), bindingData.favorViewModel);
+
 	if(!bindingInfo) {
 		return;
 	}
+
+	if(
+		bindingInfo.child === "viewModel" &&
+		!isBindingsAttribute(bindingInfo.bindingAttributeName)
+	) {
+		var name = node.nodeName || node.name;
```

Comment:
Do some Nodes not have `nodeName` and `nodeValue`? Are there tests for these different cases?

Reply:
In `VDOM`, nodes have `name` and `value` instead. Since the tests run in both `DOM` and `VDOM`, this is tested.